dashboard-widgets.el

```diff
@@ -99,6 +99,13 @@ The format is: 'icon title help action face prefix suffix'.
 Example:
 '((\"☆\" \"Star\" \"Show stars\" (lambda (&rest _) (show-stars)) 'warning \"[\" \"]\"))")
 
+(defvar dashboard-org-agenda-categories nil
```

Comment:
Since `dashboard-org-agenda-categories` its something that the user may need to customize, I think it should be a `defcustom`.

Reply:
Yes, thank you. I have made the changes you suggested.